index.html

```diff
@@ -104,7 +102,7 @@ <h3 id="hook-three">a common place?</h3>
 
             <div class="about-button"> <!--Portfolio Button-->
               <div class="about-buttn-rect">
-                <p class="about-us-button-text"> <<< Portfolio </p>
+                <p class="about-us-button-text"> <<< Portfolio </p> <!-- -->
```

Comment:
You should never use whitespace in your markup to add spacing. `padding` is a perfect candidate for this.

```css
button {
  padding: 4px 8px;
}
```

Reply:
That might explain the trouble I had centering the text correctly.